ComfortZoneApp/app/src/main/java/com/example/comfortzone/flight/ui/CityMapViewFragment.java

```diff
@@ -84,8 +118,8 @@ public void onMapReady(@NonNull GoogleMap googleMap) {
     }
 
     private List<WeatherData> getReducedCityList() {
-        if (cityList.size() > MAX_CITIES) {
-            return cityList.subList(0, MAX_CITIES);
+        if (cityList.size() > maxCities) {
+            return cityList.subList(0, maxCities);
```

Comment:
so, what if you make maxCities -1 if it's all city? that way you know it's getting all of them